test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -110,4 +118,47 @@ describe('Your PSP Stripe page', () => {
     cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
     cy.get('span').contains('Government entity document').should('have.attr', 'href', `/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}/government-entity-document`)
   })
+
+  describe('Bank details task', () => {
+    it('should click bank details task and display bank details page correctly', () => {
+      setupYourPspStubs({
+        gatewayAccountId,
+        bankAccount: false
```

Comment:
Can we get rid of this line?
So it is consistent with the other calls to `setupYourPspStubs`.